english/more_cowbell/more_cowbell.md

```diff
@@ -4,3 +4,5 @@ He has a [pocket watch](https://www.youtube.com/watch?v=YFtHjV4c4uw) in his hand
 
 He also hands you a [weapon](https://www.youtube.com/watch?v=wCDIYvFmgW8)
 
+He doesn't care about time or weapons though. He wants to become a new type of man, one that trancends the current [reality](https://www.youtube.com/watch?v=j0MAa37aaQg) and feels time as though it is water and weapons as though they are air. 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.